dts/bindings/display/st,stm32-ltdc.yaml

```diff
@@ -64,3 +65,9 @@ properties:
   window0-y1:
     type: int
     description: Last pixel in y direction on layer 0. Defaults to height.
+
+  display-controller:
+    type: phandle
+    description: |
```

Comment:
Nit: language in descriptions are usually more passive, right? IE something like "phandle of display controller. When set, used by LTDC driver to manage display blanking configuration"